website1/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import './App.css';
+import Navbar from "./components/Navbar/Navbar";
+import Contact from "./components/Contact/Contact";
+import About from "./components/About/About";
+import FoxPic from "./components/CatFact";
+import nyc from "./images/mike-c-valdivia-kZokA2VTKn4-unsplash.jpg";
+
+class App extends Component{
+  constructor(props){
+    super(props);
+    this.state={
+      currPage: "Home"
+    }
+  }
+
+  changeCurrPage = (newCurrPage) => {
+    console.log(newCurrPage)
+    this.setState({
+      currPage: newCurrPage,
+    })
+  }
+
+  renderPage = () => {
```

Comment:
Awesome job implementing this! It looks great and works this is a really clean implementation of it